test/specs/WebexMeeting/WebexMeetingPage.js

```diff
@@ -0,0 +1,95 @@
+import {createTestUser, removeTestUser} from '@webex/test-users';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+class WebexMeetingPage {
+  constructor() {
+    this.user = null;
+  }
+
+  setUser(user) {
+    this.user = user;
+  }
+
+  get accessTokenInput() {
+    return $('#token');
+  }
+  get destinationInput() {
+    return $('#destination');
+  }
+  get saveTokenBtn() {
+    return $('#md-button-1');
+  }
+  get displayWidgetBtn() {
+    return $('#md-button-3');
+  }
+  get microphoneIcon() {
+    return $('.meeting-controls-container').$$('button')[0];
+  }
+  get videoIcon() {
+    return $('.meeting-controls-container').$$('button')[1];
+  }
+  get videoRef() {
```

Comment:
there are multiple `<video>` elements in the widget, let's make sure we target the one we want:
```js
get localVideo() { return $('.wxc-local-media > video'); }
```